### PR TITLE
Update install.md

### LINKED ISSUE #456: doc bug: possibly missing "--version 2"
I think there is a bug in the docs.

In the README it says `wsl --import NixOS $env:USERPROFILE\NixOS\ nixos-wsl.tar.gz`

but after that `wsl -d NixOS` results in a `HCS_E_CONNECTION_TIMEOUT`.

However, using `wsl --import NixOS $env:USERPROFILE\NixOS\ nixos-wsl.tar.gz --version 2` in the first place works.

### PR DESCRIPTION
**Pull request by AlisterH**
specify `--version 2` so that people don't get an unusable wsl1 install (fixes #456)

### CODE REVIEW

File: docs/src/install.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ First, [download the latest release](https://github.com/nix-community/NixOS-WSL/
 Then open up a PowerShell and run:
 
 ```powershell
-wsl --import NixOS $env:USERPROFILE\NixOS\ nixos-wsl.tar.gz
+wsl --import NixOS $env:USERPROFILE\NixOS\ nixos-wsl.tar.gz --version 2
```
Comment (SuperSandro2000) [inline docs/src/install.md:15]:
```suggestion
wsl --version 2 --import NixOS $env:USERPROFILE\NixOS\ nixos-wsl.tar.gz
```

File: docs/src/install.md [hunk 2]
```diff
@@ -12,13 +12,13 @@ First, [download the latest release](https://github.com/nix-community/NixOS-WSL/
...
 ```
 
 Or for Command Prompt:
 
 ```cmd
-wsl --import NixOS %USERPROFILE%\NixOS\ nixos-wsl.tar.gz
+wsl --import NixOS %USERPROFILE%\NixOS\ nixos-wsl.tar.gz --version 2
```
Comment (SuperSandro2000) [inline docs/src/install.md:21]:
```suggestion
wsl --version 2 --import NixOS %USERPROFILE%\NixOS\ nixos-wsl.tar.gz
```

**Comment by RossSmyth** [comment]
Are you sure this command is correct? Running this did not do anything useful. I had to remove the `--version 2` because it made WSL just print its version and exit.

```powershell
# Just prints the version
PS C:\Users\rsmyth\Downloads> wsl --version 2 --import NixOS $env:USERPROFILE\NixOS\ nixos-wsl.tar.gz
WSL version: 2.2.4.0
Kernel version: 5.15.153.1-2
WSLg version: 1.0.61
MSRDC version: 1.2.5326
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.26091.1-240325-1447.ge-release
Windows version: 10.0.19045.4780
# Actually installs
PS C:\Users\rsmyth\Downloads> wsl --import NixOS $env:USERPROFILE\NixOS\ nixos-wsl.tar.gz
Import in progress, this may take a few minutes.
The operation completed successfully.
```

**Comment by AlisterH** [comment]
Looks like the `--version 2` actually needs to be at the end of the command, as per my actual pull request.

**Comment by AlisterH** [comment]
> Looks like the `--version 2` actually needs to be at the end of the command, as per my actual pull request. Does that work for you?

Actually, I think it just needs to be after the `--import`.
Can't test here right at the moment.

**Comment by SuperSandro2000** [comment]
please double check #531

**Comment by AlisterH** [comment]
Yes, that is OK.

**Pull request merged**